backend/game.py

```diff
@@ -151,3 +108,35 @@ def start_main_loop(lobby: Lobby) -> None:
     """Start the main game loop."""
     loop = GameLoop(lobby)
     threading.Thread(target=loop.run).start()
+
+
+def get_orders(day: int, num_players: int) -> Queue[Order]:
+    """Return a queue of orders for the next day."""
+    orders = Queue()
```

Comment:
just use deque or honestly even a list. much more ergonomic. we don't need the thread safety